src/Test/Utilities/Portable/Assert/AssertEx.cs

```diff
@@ -265,6 +265,98 @@ private static bool SequenceEqual<T>(IEnumerable<T> expected, IEnumerable<T> act
             return true;
         }
 
+        public static void SetEqual(IEnumerable<string> expected, IEnumerable<string> actual, IEqualityComparer<string> comparer = null, string message = null, string itemSeparator = "\r\n", Func<string, string> itemInspector = null)
+        {
+            var indexes = new Dictionary<string, int>();
```

Comment:
Shouldn't this take `comparer` as a parameter (after calculating a default value when null).